app/actions/GovernanceActions.js

```diff
@@ -9,6 +9,7 @@ import { hexReversedHashToArray, reverseRawHash } from "helpers";
 // enum values from politeiawww's v1.PropVoteStatusT
 export const VOTESTATUS_ACTIVEVOTE = 3;
 export const VOTESTATUS_VOTED = 4;
+export const VOTESTATUS_ABANDONED = 0;
```

Comment:
Is this right? On politeiawww "abandoned" is defined as having status === 6:

https://github.com/decred/politeia/blob/master/politeiawww/api/v1/v1.go#L198

Reply:
Hrm yah looks like it should be 6, but for whatever reason I am getting 0 back from the API.  Let me confirm.

Reply:
So getVetted is giving me them properly with 6, but then they aren't included in voteStatus response which results in a 0 instead.